src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.NonUap.cs

```diff
@@ -6,8 +6,63 @@
 
 namespace System.Diagnostics
 {
+    /// <summary>Provides access to local and remote processes and enables you to start and stop local system processes.</summary>
+    /// <remarks><format type="text/markdown"><![CDATA[
```

Comment:
@gewarren here's an example of a large remarks section that should be moved to an md file. I'll do that next.

Reply:
I opened a PR in dotnet-api-docs, where I'm introducing the md files (since they will live in that repo). After we merge that, I'll re-run the tool to update these comments.
https://github.com/dotnet/dotnet-api-docs/pull/5360